--- bazarr.py	(revision 7c061874fb52420b3dc39696d296a338fe4c3f88)
+++ bazarr.py	(revision 6c0d2926b8679874ae3e4535ff40ca5037f26a87)
@@ -122,21 +122,12 @@
 def start_bazarr(process_registry=ProcessRegistry()):
     script = [sys.executable, "-u", os.path.normcase(os.path.join(dir_name, 'bazarr', 'main.py'))] + sys.argv[1:]
     
-    ep = sp.Popen(script, stdout=sp.PIPE, stderr=sp.STDOUT, stdin=sp.PIPE)
-    process_registry.register(ep)
     print("Bazarr starting...")
+    ep = sp.Popen(script, stdout=None, stderr=None, stdin=sp.DEVNULL)
+    process_registry.register(ep)
     try:
-        while True:
-            line = ep.stdout.readline()
-            if line == '' or not line:
-                # Process ended so let's unregister it
-                process_registry.unregister(ep)
-                break
-            if PY3:
-                sys.stdout.buffer.write(line)
-            else:
-                sys.stdout.write(line)
-            sys.stdout.flush()
+        ep.wait()
+        process_registry.unregister(ep)
     except KeyboardInterrupt:
         pass
 
