--- bazarr/main.py	(revision 7c061874fb52420b3dc39696d296a338fe4c3f88)
+++ bazarr/main.py	(revision 908fb559401d0d69995d614af6cb69d4b9c391d5)
@@ -205,6 +205,9 @@
 @custom_auth_basic(check_credentials)
 def shutdown():
     authorize()
+    doShutdown()
+
+def doShutdown():
     try:
         server.stop()
     except:
@@ -2215,4 +2218,4 @@
         args.port) if args.port else str(settings.general.port)) + str(base_url))
     server.start()
 except KeyboardInterrupt:
-    shutdown()
+    doShutdown()
